README.md

```diff
@@ -1,26 +1,29 @@
-# Large Tether Transfer Agent
+# New bot deployment detector
 
 ## Description
 
-This agent detects transactions with large Tether transfers
+This bot detects new bots deployed from the Nethermind teams forta-deployer address
 
 ## Supported Chains
 
-- Ethereum
-- List any other chains this agent can support e.g. BSC
+- Polygon
 
 ## Alerts
 
-Describe each of the type of alerts fired by this agent
-
-- FORTA-1
-  - Fired when a transaction contains a Tether transfer over 10,000 USDT
-  - Severity is always set to "low" (mention any conditions where it could be something else)
-  - Type is always set to "info" (mention any conditions where it could be something else)
-  - Mention any other type of metadata fields included with this alert
+- NETH-1
+  - Fired when a transaction from the Forta deployer address creates a new bot.
+  - Severity is always set to "info"
+  - Type is always set to "info"
+  - Metadata:
+    - agentId: unique id of the bot
+    - metadata: metadata of the bot
+    - chainId: chains supported by the bot
 
 ## Test Data
 
-The agent behaviour can be verified with the following transactions:
+The bot behaviour can be verified with the following transactions:
+
+- (create bot function)
+  - tx: 0xb9fffc84511a7135aeed7db1f560951dac6c9ea49c8de353e9c41d70a322b210
```

Comment:
```suggestion
  - tx: 0xb9fffc84511a7135aeed7db1f560951dac6c9ea49c8de353e9c41d70a322b210
  -tx: [0xb9fffc84511a7135aeed7db1f560951dac6c9ea49c8de353e9c41d70a322b210](https://www.polygonscan.com/tx/0xb9fffc84511a7135aeed7db1f560951dac6c9ea49c8de353e9c41d70a322b210)
```

We usually write the example in a way that it includes the block explorer link, like explained [here](https://www.notion.so/nethermind/Definition-of-Done-Guidelines-6d81308092214ec6a0d13089127c87f9#76965a627cbd46a2bf2258c494e16434)